include/zephyr/net/net_ip.h

```diff
@@ -745,6 +745,38 @@ static inline bool net_ipv6_is_prefix(const uint8_t *addr1,
 	return (addr1[bytes] & mask) == (addr2[bytes] & mask);
```

Comment:
Please change the commit subject of the 1st one to `net: Add ipv6 prefix mask function`